src/app/workbasket/workbasket.component.ts

```diff
@@ -12,6 +12,8 @@ import { CaseType } from '../shared/domain/definition/case-type.model';
 import { FormGroup } from '@angular/forms';
 import { SearchResultComponent } from '../shared/search/search-result.component';
 import { Observable } from 'rxjs/Observable';
+import { AlertService } from '../core/alert/alert.service';
+import { plainToClass } from 'class-transformer';
```

Comment:
this class too could use some refactoring sooner or later

Reply:
this code fragment is from another ticket and got merged here accidentally